modules/hosted-control-planes-version-support.adoc

```diff
@@ -31,10 +52,25 @@ You can host different versions of control planes on the same management cluster
       namespace: hypershift
 ----
 
-You can use the `hcp` CLI to create hosted clusters.
+[id="hcp-versioning-cli_{context}"]
+== {hcp} CLI
+
+You can use the `hcp` CLI to create hosted clusters. You can download the CLI from {mce-short}. When you run the `hcp version` command, the output shows the latest {product-title} that the CLI supports against your `kubeconfig` file.
+
+[id="hcp-versioning-api_{context}"]
+== hypershift.openshift.io API
 
 You can use the `hypershift.openshift.io` API resources, such as, `HostedCluster` and `NodePool`, to create and manage {product-title} clusters at scale. A `HostedCluster` resource contains the control plane and common data plane configuration. When you create a `HostedCluster` resource, you have a fully functional control plane with no attached nodes. A `NodePool` resource is a scalable set of worker nodes that is attached to a `HostedCluster` resource.
 
 The API version policy generally aligns with the policy for link:https://kubernetes.io/docs/reference/using-api/#api-versioning[Kubernetes API versioning].
 
-Updates for {hcp} involve updating the hosted cluster and the node pools. For more information, see "Updates for hosted control planes".
\ No newline at end of file
+Updates for {hcp} involve updating the hosted cluster and the node pools. For more information, see "Updates for hosted control planes".
+
+[id="hcp-versioning-cpo_{context}"]
+== Control Plane Operator
+
+The Control Plane Operator is released as part of each {product-title} payload release image:
```

Comment:
```suggestion
The Control Plane Operator is released as part of each {product-title} payload release image for the following architectures:
```
or
```suggestion
The Control Plane Operator is released as part of the {product-title} payload release image for each architecture:
```
or similar? Probably also clear enough as is.

Reply:
I went with your first suggestion.